src/server/common/model/answer/email/email-address.js

```diff
@@ -1,9 +1,32 @@
 import { config } from '~/src/config/config.js'
 import { TextAnswer } from '../text/text.js'
+import fs from 'fs'
+import path from 'path'
 
 /** @import {TextConfig} from '../text/text.js' */
 
-const emailAddressRegex = /^[^@]+@[^@]+$/
+const tldsPath = path.resolve(
+  path.dirname(new URL(import.meta.url).pathname),
+  'tlds.txt'
+)
+let tldPattern = ''
+try {
+  const tlds = fs
+    .readFileSync(tldsPath, 'utf-8')
+    .split('\n')
+    .map((line) => line.trim())
+    .filter((line) => line && !line.startsWith('#'))
+    .join('|')
+  tldPattern = tlds
+} catch (e) {
+  // fallback to a generic pattern if file read fails
```

Comment:
I think I'd rather let it crash.  In CDP, a service that is dead on arrival will fail its `/health` endpoint checks & the deploy will fail without traffic being routed to it.